File: javarecipe/src/main/java/work/shion/javarecipe/pages/entrypoint/contracts/TabPresenterContract.java
package work.shion.javarecipe.pages.entrypoint.contracts;


import android.view.MenuItem;

/**
 * タブの挙動定義
 */
public interface TabPresenterContract {

    /**
     * NavigationItem 選択時
     */
    boolean onNavigationItemSelected(MenuItem selected);
}

File: ktrecipe/src/main/java/work/shion/ktrecipe/pages/entrypoint/contracts/TabPresenterContract.kt
package work.shion.ktrecipe.pages.entrypoint.contracts

import android.view.MenuItem


/**
 * タブの挙動定義
 */
interface TabPresenterContract {

    /**
     * NavigationItem 選択時
     */
    fun onNavigationItemSelected(selected: MenuItem): Boolean
}
